akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEventParser.scala

```diff
@@ -22,6 +22,61 @@ import scala.util.control.NonFatal
 
 private object ServerSentEventParser {
 
+  private final class Builder {
+
+    private var data = Vector.empty[String]
```

Comment:
I recommend switching to a mutable data structure for this now, as it is a mutable builder anyway.